.travis.yml

```diff
@@ -1,14 +1,43 @@
 dist: trusty
 sudo: required
+
 services:
-- docker
+  - docker
+
 before_install:
-- openssl aes-256-cbc -K $encrypted_fe7d4190ec44_key -iv $encrypted_fe7d4190ec44_iv
-  -in id_rsa.travis.enc -out ~/.ssh/id_rsa -d
-- chmod 600 ~/.ssh/id_rsa
-- sudo sysctl -w vm.max_map_count=262144
-script:
-- bash test/run.sh
+  - openssl aes-256-cbc -K $encrypted_fe7d4190ec44_key -iv $encrypted_fe7d4190ec44_iv -in id_rsa.travis.enc -out ~/.ssh/id_rsa -d
+  - chmod 600 ~/.ssh/id_rsa
+  - sudo sysctl -w vm.max_map_count=262144
+
+stages:
+  - name: test
+
+jobs:
+  include:
+    - stage: test
+      name: fedora 28
```

Comment:
Nitpicking: In the other repositories Travis configuration, I capitalize stage name.